FRAuth/FRAuth/DeviceBinding/ApplicationPinDeviceAuthenticatorDelegate.swift

```diff
@@ -0,0 +1,72 @@
+// 
+//  ApplicationPinDeviceAuthenticatorDelegate.swift
+//  FRAuth
+//
+//  Copyright (c) 2023 ForgeRock. All rights reserved.
+//
+//  This software may be modified and distributed under the terms
+//  of the MIT license. See the LICENSE file for details.
+//
+
+
+import UIKit
+
+
+/// ApplicationPinDeviceAuthenticatorDelegate for collecting the Pin
+public protocol ApplicationPinDeviceAuthenticatorDelegate: AnyObject {
```

Comment:
To avoid confusion about this protocol I would suggest renaming this to 
`public protocol ApplicationPinDeviceAuthenticatorCollectorDelegate: AnyObject {`

Reply:
Maybe CredentialsCollector?

```
CredentialsCollector {
    onCredentialsRequest()
}
```

Reply:
Basically this is a view delegate ,,,so i am thinking like like CredentialsCollectorViewDelegate  or PinCollectorViewDelegate and actual class name like PinCollector or Credentialcollector

Reply:
We had a meeting with the team and decided to go with **PinCollector** name. Aligned with Android as well.